packages/tracecat-registry/tracecat_registry/core/cases.py

```diff
@@ -748,6 +754,147 @@ async def remove_case_tag(
         await service.tags.remove_case_tag(case.id, tag)
 
 
+@registry.register(
+    default_title="Link table row to case",
+    display_group="Cases",
+    description="Link an existing table row to a case.",
+    namespace="core.cases",
+)
+async def link_case_table_row(
```

Comment:
Prefer more generalized CRUD terminology e.g. `create`, `add` etc. even though it's a "link"

```suggestion
async def add_rows(
```